src/plugins/timelist/Timelist.vue

```diff
@@ -144,8 +152,17 @@ export default {
         if (this.clearAutoScrollDisabledTimer) {
             clearTimeout(this.clearAutoScrollDisabledTimer);
         }
+
+        this.composition.off('add', this.addToComposition);
```

Comment:
I think you want to check for composition here before unlistening, like you're doing when you listen.